liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -511,24 +510,13 @@ private String getEEVersion(MavenProject project) throws NoUmbrellaDependencyExc
                     continue;
                 }
                 if (d.getGroupId().equals("javax") && d.getArtifactId().equals("javaee-api")) {
-                    if (d.getVersion().startsWith("8.")) {
-                        return BINARY_SCANNER_EEV8;
-                    } else if (d.getVersion().startsWith("7.")) {
-                        return BINARY_SCANNER_EEV7;
-                    } else if (d.getVersion().startsWith("6.")) {
-                        return BINARY_SCANNER_EEV6;
-                    }
+                    return composeEEVersion(d.getVersion());
                 } else if (d.getGroupId().equals("jakarta.platform") &&
```

Comment:
Would it be better to do an "or" statement here instead of an "else if" since it's the same line?
```
if (d.getGroupId().equals("javax") && d.getArtifactId().equals("javaee-api")) || (d.getGroupId().equals("jakarta.platform") && d.getArtifactId().equals("jakarta.jakartaee-api")) {
   return composeEEVersion(d.getVersion());
}
```

Reply:
I just didn't want the conditional to get too long but this looks ok.